README.md

```diff
@@ -89,7 +89,8 @@ usage is below threshold for all (cpu, memory, and number of pods), the node is
 Currently, pods request resource requirements are considered for computing node resource utilization.
 
 There is another configurable threshold, `targetThresholds`, that is used to compute those potential nodes
-from where pods could be evicted. Any node, between the thresholds, `thresholds` and `targetThresholds` is
+from where pods could be evicted. If a node's usage is above targetThreshold for any (cpu, memory, or number of pods),
+the node is considered over utilized. Any node, between the thresholds, `thresholds` and `targetThresholds` is
```

Comment:
Any node between the thresholds `thresholds` and `targetThresholds` is ... (no need for the comas)